ansible/playbooks/roles/repository/tasks/RedHat/install-packages.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- include_vars:
+    file: roles/common/vars/main.yml
+    name: common_vars
+
+# Fix to solve issues with expiring certificates on Azure
+# https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-rhui#update-expired-rhui-client-certificate-on-a-vm
+# TODO: Needs to be updated for RHEL 8
+- name: Update repository certificates
+  command: yum update -y --disablerepo='*' --enablerepo='*microsoft*'
```

Comment:
Change to yum module - https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#parameter-update_only

Reply:
Done.